litellm/llms/prompt_templates/factory.py

```diff
@@ -1342,9 +1342,14 @@ def prompt_factory(
             return hf_chat_template(
                 model=model, messages=messages, chat_template=chat_template
             )
+        elif "command-r" in model:
```

Comment:
Hi @nalf3in, 

When would this be triggered? 

our hf logic should already be handling the templating for this if you call it via `huggingface/CohereForAI/c4ai-command-r-plus` or any other CohereForAI/c4ai-command-r.. implementation

see `hf_chat_template`